### PR TITLE
Es6 template literals

### PR DESCRIPTION
**Pull request by gquathamer**
image

### CODE REVIEW

File: es6-template-literals/index.html [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    const age = 25;
+
+    const bio = `My name is ${firstName} ${lastName} and I am ${age} years old.`;
+
+    console.log('Bio:', bio);
+
+    const fruit = 'apple';
+    const item = 'pen';
+
+    const applePen = fruit + item;
```
Comment (TimelikeClosure) [inline es6-template-literals/index.html]: You missed a concatenation

**Review by TimelikeClosure — CHANGES_REQUESTED** [review]

File: es6-template-literals/index.html [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    const age = 25;
+
+    const bio = `My name is ${firstName} ${lastName} and I am ${age} years old.`;
+
+    console.log('Bio:', bio);
+
+    const fruit = 'apple';
+    const item = 'pen';
+
+    const applePen = `${fruit} ${item}`;
```
Comment (TimelikeClosure) [inline es6-template-literals/index.html]: Please don't add or remove characters while refactoring:

---

image

**Review by TimelikeClosure — CHANGES_REQUESTED** [review]

**Comment by gquathamer** [comment]
image

**Comment by gquathamer** [comment]
image

**Review by TimelikeClosure — APPROVED** [review]

**Pull request merged**